docker/dockerfile-build-centos

```diff
@@ -37,8 +35,8 @@ RUN echo '#!/bin/bash' | tee /etc/profile.d/devtoolset7.sh && echo \
 # docker pipeline runs containers with particular uid
 # create a jenkins user with this specific uid so it can use sudo priviledges
 # Grant any member of sudo group password-less sudo privileges
-RUN useradd --create-home -u ${user_uid} -o -G wheel --shell /bin/bash jenkins && \
-    echo '%wheel ALL=(ALL) NOPASSWD:ALL' | tee /etc/sudoers.d/sudo-nopasswd && \
+RUN useradd --create-home -u ${user_uid} -o -G video --shell /bin/bash jenkins && \
+    echo '%video ALL=(ALL) NOPASSWD:ALL' | tee /etc/sudoers.d/sudo-nopasswd && \

```

Comment:
why did you need the video group?

Reply:
There was an error complaining about not having access to the GPU. I thought that meant that a non-sudo user couldn't access the GPU, so I switched this to wheel. That then caused permissions issues when running tests. I'm switching it back to the video group because the develop branch is passing CI right now, and the user is being added to the video group there.